lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "send_sms"
+require 'dotenv/load'
+
+class Order
+  def initialize(dish, quantity = 1)
+    @selected_dishes = []
+    quantity.times { @selected_dishes.push(dish) }
+    @confirmed = false
+    view_total
+  end
+
+  def update(dish, quantity = 1)
+    quantity.times { @selected_dishes.push(dish) }
+    view_total
+  end
+
+  def view_selected
+    @selected_dishes
+  end
+
+  def view_total
+    total = 0
+    @selected_dishes.each do |dish|
+      total += dish.price
+    end
+    @total = total
+  end
+
+  def confirm_order(
+    _to = ENV['TWILIO_TO_NUMBER'],
+    message_class = Sms,
+    time = calculate_time
+  )
+    sms = message_class.new
+    @confirmed = true
```

Comment:
Do you check the total of the order against a passed-in total? "Place the order by giving the list of dishes, their quantities and a number that should be the exact total. If the sum is not correct the method should raise an error"